app/src/main/java/io/appium/uiautomator2/handler/FindElements.java

```diff
@@ -157,38 +157,31 @@ public AppiumResponse safeHandle(IHttpRequest request) {
         if (element == null) {
             throw new ElementNotFoundException();
         }
-        if (by instanceof By.ById) {
-            String locator = getElementLocator((ById) by);
-            return getInstance().findObjects(android.support.test.uiautomator.By.res(locator));
+        if (by instanceof ById) {
+            String locator = getElementLocator((ById)by);
+            return element.getChilds(android.support.test.uiautomator.By.res(locator));
```

Comment:
Is `getChilds` actually the name of the method? Do we have control over that? If we do, that should be `getChildren`.